hoth_api.yaml

```diff
@@ -319,5 +319,351 @@ components:
       oneOf:
         - $ref: "#/components/schemas/DescriptionOfWork"
     DescriptionOfWork:
-      description: "TODO: AT-7341, define DescriptionOfWork schema"
+      description: "Schema used to generate DescriptionOfWork document"
       type: object
+      properties:
+        to_title:
+          type: string
+          maxLength: 60
+        scope:
+          type: string
+          maxLength: 300
+        scope_surge:
+          type: integer
+          maximum: 50
```

Comment:
I think data related to the contract award (`Award/Mod Effective` and `Version Date` columns) is missing, which is used for the first table in the DoW [see attachment in epic](https://ccpo.atlassian.net/browse/AT-7232).